### ISSUE TITLE
Python 3 version ?

### ISSUE THREAD

**Issue by rwest** [ReactionMechanismGenerator/PyDAS#14]
See also #12 

@mliu49 could you share what you've done successfully and where the problem is on Mac?

**Comment by mliu49** [comment]
There is a py3 branch which has the changes I made to successfully build on Linux.

I will push a couple additional changes I made for Mac and comment with the error I encountered once I get home.

**Comment by mliu49** [comment]
I am very confused right now. I tried building the Conda package again, and it failed the first time because the UCSB website for downloading DASPK is down. Then I changed it to get it from a local copy I had from before and the package compiled successfully and passed all of the tests.

The last time I tried, it definitely failed on the testing stage. I don't think I've changed anything in my environment, but maybe some dependency changed in the meantime?

I pushed the changes I had to the py3 branch. The main one is having Conda handle the fortran compilers automatically. It is a bit messy though since I just commented stuff out.

I would like to test the changes to the fortran compiler setup on linux again, and the clean up the changes and push to master.

The UCSB website being down is problematic though. I'm not too familiar with the licensing, but I assume we can't include the source code with PyDAS? Perhaps we can just adjust the conda build to use a local copy but not include it in the final package.
